static/styles/left-sidebar.css

```diff
@@ -190,6 +190,7 @@ ul.filters .arrow {
     position: absolute;
     right: 0px;
     top: 2px;
+    right: 4px;
```

Comment:
5px was keeping the chevron too close to the count div. 4px places it right between the scrollbar and the counter.